full/src/main/java/apoc/custom/CypherProcedures.java

```diff
@@ -104,7 +101,7 @@ public void asFunction(@Name("signature") String signature, @Name("statement") S
                 ProcedureSignature signature = procedureDescriptor.getSignature();
                 return new CustomProcedureInfo(
                         PROCEDURE,
-                        signature.name().name(),
+                        signature.name().toString().replace(PREFIX + ".", ""),
```

Comment:
Invece che replace visto che qualcuno potrebbe usare `custom.` nel nome tipo `custom.custom.my_name` fai:
```suggestion
                        signature.name().toString().substring(PREFIX.length + 1),
```